release 2.25.1 (#1057)
microsoft/onefuzz: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## 2.25.1
### Fixed

> * Service: Fixed an issue recovering nodes that fail setup correctly.  [#1054](https://github.com/microsoft/onefuzz/pull/1054)

Reviewer: ```suggestion
* Service: Fixed an issue detecting and reimaging failed nodes.  [#1054](https://github.com/microsoft/onefuzz/pull/1054)
```
